packages/shopping/src/controllers/user.controller.ts

```diff
@@ -58,18 +74,37 @@ export class UserController {
       },
     },
   })
-  async create(@requestBody() user: User): Promise<User> {
+  async create(
+    @requestBody({
+      content: {
+        'application/json': {
+          schema: getModelSchemaRef(NewUserRequest, {
+            title: 'NewUser',
+            exclude: ['id'],
```

Comment:
Why are we excluding id?

Reply:
In the API explorer, we had to choice of passing or not passing an `id` value when creating a user

Reply:
Good point.

In a typical application, the id is generated by the database. That's how we are scaffolding new models via `lb4 model` and `lb4 controller` too. 

I'll double check what's the actual setup in this example app and ensure the controller is consistent with model.

Reply:
redis keeps failing on my local I couldn't try creating a user with explorer, but `forceId` should be default to `true`, so I think excluding the id field is correct here.

Reply:
I believe that `forceId` is enabled only when the `id` property is defined with `generated: true`, which is not the case in our Shopping service - see here:

https://github.com/strongloop/loopback4-example-shopping/blob/19d6c9265e46eb185c8600617134835985797082/packages/shopping/src/models/user.model.ts#L25-L29

I am going to remove `exclude: ['id']` for consistency with other parts of the codebase. We can revisit this aspect later, it's out of scope of my pull request.

Reply:
In the shopping app, is has always been historically possible to enter an id for a new User when posting to /users. And it has historically been possible to avoid specifying an id (so that it is automatically generated) when creating a user. Let's make sure this still works. So let's be sure not to modify any properties if the id field. thx. Please confirm this still works.

Reply:
@bajtos ^

Reply:
> redis keeps failing on my local I couldn't try creating a user with explorer, but `forceId` should be default to `true`, so I think

that is weird.

You may need to run:

docker volume prune
docker images prune
docker prune

to clean up docker.

Usually if a container isn't running when it should.... a clean up like this fixes it.

Reply:
> In the shopping app, is has always been historically possible to enter an id for a new User when posting to /users. And it has historically been possible to avoid specifying an id (so that it is automatically generated) when creating a user. Let's make sure this still works. So let's be sure not to modify any properties of the id field. thx. Please confirm this still works.

All existing tests are passing. AFAICT, they are usually not providing any `id` value and relying on the database to generate one.

As for providing a custom id value, either it's covered by tests and then it is still working (because all tests are passing), or else the behavior is undefined as far as I am concerned in this pull request 🤷‍♂

Reply:
Please ignore my comment above, I'll investigate this use case based on what we have in our documentation, as discussed elsewhere in this PR.